src/components/Footer/Footer.tsx

```diff
@@ -1,15 +1,21 @@
 /* eslint-disable @next/next/no-img-element */
-import { Row, Col, Layout } from "antd";
+import { Col, Layout, Row } from "antd";
 import { useTranslation } from "next-i18next";
 import React from "react";
-import { useMediaQuery } from "../../hooks/useMediaQuery";
+
+import { useAppSelector } from "../../store/store";
 import colors from "../../styles/colors";
 import AletheiaSocialMediaFooter from "./AletheiaSocialMediaFooter";
 import FooterInfo from "./FooterInfo";
 
 const Footer = () => {
     const { t } = useTranslation();
-    const isMobile = useMediaQuery("(max-width: 767px)");
+    const { isMobile } = useAppSelector((state) => {
```

Comment:
Looking good!

Maybe it will be more semantic if we stop using `isMobile` and start using the breakpoints directly, what do you think? Here's an example:

```
const layout = useAppSelector(state => state.breakpoints)
[...]
<Row
                        style={{
                            marginTop: "10px",
                            width: "100%",
                            textAlign: "center",
                            flexDirection: "column",
                            marginBottom: layout.sm ? "32px" : "0",
                        }}
                    >
[...]
```

Reply:
I'm also okay with naming this new variable to `vw` as in video width, since this is related to CSS as long as we document that in the code.